src/package.ts

```diff
@@ -85,4 +93,26 @@ export class Package {
   version(): string {
     return this.packageURL.version || ''
   }
+
+  /**
+   * Provided a "matcher" object with any of the string fields 'namespace',
+   * 'name', or 'version', returns true if the Package has values matching the
+   * matcher.
+   *
+   * @param {Object} matcher
+   * @returns {boolean}
+   */
+  matching(matcher: {
+    namespace?: string
+    name?: string
+    version?: string
+  }): boolean {
+    return (
+      (matcher.namespace === undefined ||
+        this.packageURL.namespace === matcher.namespace) &&
+      (matcher.name === undefined || this.packageURL.name === matcher.name) &&
```

Comment:
this would be slightly more readable if you indented each clause the same way

Reply:
I agree. I was fighting with the auto-formatter (`prettier`, invoked by `npm run format` and `npm run all`). The auto-formatter won 😥

Reply:
fixed with an ignore line
```
// prettier ignore
```